model/project_ref_test.go

```diff
@@ -3277,6 +3351,29 @@ func TestSaveProjectPageForSection(t *testing.T) {
 	assert.Equal(projectRef.ProjectHealthView, ProjectHealthViewAll)
 	assert.True(utility.FromBoolPtr(projectRef.Restricted))
 	assert.True(utility.FromBoolPtr(projectRef.Private))
+
+	// Test GitHub token permissions by requester being set
+	update = &ProjectRef{
```

Comment:
Can we add some validation that the requester is a requester that exists? I don't expect us to run into this because the UI will only allow selecting from a list, but it's good to have in case this is ever expanded.